cmd/config/metrics-profiles/metrics.yml

```diff
@@ -187,3 +187,13 @@
 
 - query: rate ( node_vmstat_pgmajfault[1m] )
   metricName: nodeMajorFaults
+
+# OVS instantaneous per-second rate of increase over the last 2 minutes
+- query: sum (  irate( container_cpu_usage_seconds_total {  id =~ "/system.slice|/system.slice/kubelet.service|/system.slice/ovs-vswitchd.service|/system.slice/crio.service|/kubepods.slice" }[2m])  )  by   (   id , instance )
+  metricName: cgroupOVSCPUUsagePercent
+
+# OVS RSS Memory 
+- query: sum(container_memory_rss{id=~"/system.slice/ovs-vswitchd.service"}) by (id)
```

Comment:
Similar to above. I see this as: "You want the time-series resolution of ovs-vswitchd". Let's keep it consistent by capturing all the same cgroup id's from the other queries, and make the metricName more generic by removing "OVS".